GOVERNANCE.md

```diff
@@ -0,0 +1,63 @@
+# Project Governance
+*Last updated June 9, 2022*
+
+## Overview
+
+FOSSBilling is an open source project which builds and maintains a free, open source billing and client management solution. 
+
+Anybody with an interest can join the community, contribute to the project, and be a part of the decision making process. In the next few paragraphs we will explain how that process takes place. 
+
+__TLDR; Project decisions are made by a consensus of the projects steering committee and informed by community discussion.__
+
+## Community
+
+The FOSSBilling community is made up of individuals, companies, or groups who use and/or contribute to the FOSSBilling software.
+
+Anybody can use the core FOSSBilling software within the terms of the [license](https://github.com/FOSSBilling/FOSSBilling/blob/main/LICENSE). There are not and never will be any fees for usage, and no special requirements.
+
+We really appreciate user contributions but if you just want to download, install, and go, then that is also fine. If you do want to be more involved in the community then [GitHub](https://github.com/FOSSBilling/FOSSBilling) and [Discord](https://fossbilling.org/discord) are good places to start. 
+
+## Steering Committee
+
+The steering committee is made up of community members and developers who make a sustained contribution to, and are recognised as being interested in, the long term health of the FOSSBilling project. 
+
+Each member of the steering committee also takes ownership of the project on GitHub. This ensures that in the event of any one person abandoning FOSSBilling or being unavailable for some reason the project does not stagnate or stop.
+
+At this moment the Steering Committee includes six members - identified here by their GitHub usernames:
+
+* [0x6b.dev](https://github.com/0x6b-dev)
+* [andpavlenko](https://github.com/andpavlenko)
+* [BelleNottelling](https://github.com/BelleNottelling)
+* [Evrifaessa](https://github.com/evrifaessa)
+* [HunterNyan](https://github.com/HunterNyan)
+* [jaapmarcus](https://github.com/jaapmarcus)
+
+Members of the steering committee may recommend and appoint new members from the community at any time. A nomination for a new member will result in a discussion and a vote by the existing committee members. 
+
+Members may be removed from the committee if they resign, or if they do not participate in the community or contribute to the project for a period of 6 months. 
+
+## Decision making process
+
+Major decisions about the future of FOSSBilling will be made after community discussion and by consensus of the steering committee. 
```

Comment:
Maybe we need a separate repo for recording major resolutions (with discussion points) for transparency reasons.

Reply:
I'm not sure that's necessary, but also not against it. Anybody else have an opinion.